enhancements/monitoring/monitoring-config.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Configuration CRD for the Cluster Monitoring Operator
+authors:
+  - "@marioferh"
+  - "@danielmellado"
+  - "@jan--f"
+  - "@moadz"
+  - "simonpasquier"
+reviewers:
+  - "@jan--f"
+  - "@moadz"
+  - "simonpasquier"
+approvers:
+  - "@jan--f"
+  - "@moadz"
+  - "simonpasquier"
+  - "@openshift/openshift-team-monitoring"
+creation-date: 2024-04-26
+last-updated: 2024-04-26
+status: provisional
+---
+
+# CRD ClusterMonitoring
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+* Currently, the monitoring stack is configured using a configmaps. In OpenShift though the best practice is to configure operators using custom resources.
+
+
+## Motivation
+
+* The specification is well known and to a degree self-documenting
+* The APIServer will validate user resources based on our specifications, so users get immediate feedback on errors instead of having to check if their config was applied and check logs.
+* Many users expect to interact with operators through a CRD
+* Compatible with GitOps workflows. 
+* We can add [cross]validation rules to CRD fields to avoid misconfigurations
+* End users get a much faster feedback loop. No more applying the config and scanning logs if things don't look right. The API server will give immediate feedback
+* Organizational users (such as ACM) can manage a single resource and observe its status
```

Comment:
We can link to https://issues.redhat.com/browse/ACM-2545. It can also be merged with the GitOps argument since it's kind of the same request.
```suggestion
* CRDs supports multiple actors managing the same resource which is a key property for the Observability service of Advanced Cluster Management.
```